langchain/utilities/serpapi.py

```diff
@@ -143,6 +143,11 @@ def _process_response(res: dict) -> str:
             and "game_spotlight" in res["sports_results"].keys()
         ):
             toret = res["sports_results"]["game_spotlight"]
+        elif (
```

Comment:
@hwchase17  / @vowelparrot  this looks good to me. 

Stepping back, do we need to give the user more control in the init the user to limit the types of results that are returned. For example, for a shopping agent, a tool that only returns shopping results might work better.

Reply:
thats a great idea for a future extension!

Reply:
Ya  - the ordering of these fallbacks is also not universal I'd imagine